arlas-rest/src/main/java/io/arlas/server/rest/explore/search/GeoSearchRESTService.java

```diff
@@ -487,6 +498,7 @@ public Response geosearchPost(
         request.headerRequest = searchHeader;
```

Comment:
I think we should add the fields in `column-filter` into the `include` parameter